.github/workflows/ci.yaml

```diff
@@ -35,6 +35,12 @@ jobs:
 
       - uses: actions/checkout@v3
 
+      - name: Install dev dependancies
+        run: |
+          python -m pip install '.[dev]'
+          mypy --install-types
+
+
```

Comment:
You can delete this since run-tox won't use this installation now that tox is properly installing the dev extra.

Reply:
ok will try